core/src/main/java/org/apache/iceberg/MetadataUpdate.java

```diff
@@ -343,6 +364,11 @@ public Map<String, String> updated() {
     public void applyTo(TableMetadata.Builder metadataBuilder) {
       metadataBuilder.setProperties(updated);
     }
+
+    @Override
+    public void applyTo(ViewMetadata.Builder viewMetadataBuilder) {
+      viewMetadataBuilder.properties(updated);
```

Comment:
`setProperties`? I think the issue with these method names is that there is no verb, so it isn't clear what is happening.

Reply:
right now all the builder methods are named exactly like in the generated Builder. I'll go ahead and update them to have `setXyz` so that we're in line with the Builder from `TableMetadata`

Reply:
done